### PR TITLE
Bug ext vortex shedding datapipe

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Description

Fixing a small bug in the vortex shedding dataloader

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is
up to date with these changes.

### CODE REVIEW

**Review by ktangsali — APPROVED** [review]

File: modulus/datapipes/gnn/__init__.py [hunk 1]
```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 # from .vortex_shedding_dataset import VortexSheddingDataset
-from .ahmed_body_dataset import AhmedBodyDataset
+# from .ahmed_body_dataset import AhmedBodyDataset
```
Comment (NickGeneva) [inline modulus/datapipes/gnn/__init__.py:16]: Is there a reason to keep these here as commented out?

**Review by loliverhennigh — APPROVED** [review]

File: modulus/datapipes/gnn/__init__.py (hunk 1 above)
Comment (mnabian) [reply modulus/datapipes/gnn/__init__.py]: Not really. I'll just remove them

**Pull request merged**